vutil.c

```diff
@@ -704,7 +697,7 @@ Perl_upg_version(pTHX_ SV *ver, bool qv)
 #endif
 
 	if (sv) {
-                Perl_sv_catpvf(aTHX_ sv, "%.9" NVff, SvNVX(ver));
+                Perl_sv_setpvf(aTHX_ sv, "%.9" NVff, SvNVX(ver));
```

Comment:
Not sure what this change is for. Might be worth tracing in the version repo.

Reply:
https://github.com/Perl/version.pm/pull/22

Reply:
Thanks!